frontend/src/helpers/LandlordHelpers.js

```diff
@@ -0,0 +1,15 @@
+export const calculateAverageRatings = (reviews) => {
+  const repairsAverage = reviews.reduce((prev, curr) => prev + curr.repair, 0) / reviews.length;
+  const respectAverage = reviews.reduce((prev, curr) => prev + curr.respect, 0) / reviews.length;
+  const safetyAverage = reviews.reduce((prev, curr) => prev + curr.healthSafety, 0) / reviews.length;
+  const overall = (repairsAverage + respectAverage + safetyAverage) / 3;
```

Comment:
I think overallScore is already calculated per review:

```js
  const review: HydratedDocument<IReview> = new Review({
    title: title,
    description: desc,
    sentiment: sentiment,
    healthSafety: healthSafety,
    respect: respect,
    repair: repair,
    overallScore: calculatedScore,
    userId: userId,
    landlordId: landlordId,
  });
  ```
  
  so you could theoretically just use reduce again :)

Reply:
Do we have a place in backend to update overall score? If not i can add it to this PR

Reply:
only when you submit an edited review, it will recalculate the new overall score

Reply:
it's found in `ReviewService.ts` `getCalculatedReviewScore`